build/index.html

```diff
@@ -0,0 +1,316 @@
+            <ul class="tab-list">
+                      <li class="card-item price-tab fadeInRight wow" wow-delay="2s"><span class="tab-price">0 $</span>
+                        <h3 class="low-priority-title tab-title">Standart</h3>
+                        <p class="description-text">Back to the Office — free service. We develop it for ourselves and want to share it with you.</p>
+                        <button class="g-btn open-popup">apply for</button>
+                      </li>
+                      <li class="card-item price-tab fadeInRight wow" wow-delay="3s"><span class="tab-price">0 $</span>
+                        <h3 class="low-priority-title tab-title">Setting up Support</h3>
+                        <p class="description-text">We will help you to learn and configure the service for free.</p>
+                        <button class="g-btn open-popup">apply for</button>
+                      </li>
+                      <li class="card-item price-tab fadeInRight wow" wow-delay="4s"><span class="tab-price">Details</span>
+                        <h3 class="low-priority-title tab-title">Additional</h3>
+                        <p class="description-text">If you lack functionality, contact us. We will develop it and update your version of the service.</p>
+                        <button class="g-btn open-popup">contact us ⟶</button>
+                      </li>
+            </ul>
+          </div>
+        </section>
+        <section class="contacts" id="contacts">
+          <div class="container">
+            <h3 class="section-title contacts-title fadeIn wow" wow-delay="2s">Contact Us</h3>
+            <div class="contacts__content">
+              <div class="contacts__content-left fadeInLeft wow" wow-delay="2s"><span class="contacts-text">We’ll Glad to Hear From You</span>
+                <form class="contacts-form form" action="#" method="post">
+                  <div class="form__block">
+                    <label class="form__label" for="name">your name</label>
+                    <input class="form__input" type="text" name="name" id="name" required>
+                  </div>
+                  <div class="form__block">
+                    <label class="form__label" for="email">email</label>
+                    <input class="form__input email" type="email" name="email" id="email" required>
+                  </div>
+                  <label class="contacts-form__textarea-label" for="message">message</label>
+                  <textarea class="contacts-form__textarea" name="message" id="message"></textarea><input type="hidden" name="bottom-form" value="bottom-form">
+                  <input class="g-btn contacts-btn" type="submit" value="send us ⟶">
+                </form>
+              </div>
+              <div class="contacts__content-right fadeInRight wow" wow-delay="2s"></div>
+            </div><a class="write-to-us fadeIn wow" href="mailto:backtotheoffice@gmail.com" wow-delay="2s">
```

Comment:
Please change mail to `btto.contact@gmail.com`

Reply:
fixed